### DISCUSSION TITLE
11.5-alpha can't render markdown file

### DISCUSSION THREAD

**Discussion by amitbha** [orbitalquark/textadept#281] [Q&A] [answered]
![1015-1323-05](https://user-images.githubusercontent.com/4060625/195970383-bd057d7b-6880-4874-9118-38d54d5e5c00.jpg)

**Comment by orbitalquark** [comment, marked as answer]
Thanks for the report. Are you using a custom theme? If so, what is it? I'm getting Markdown highlighting with Textadept's default theme(s).

**Reply by amitbha** [reply]
Yes. I'm using `https://github.com/rgieseke/base16-textadept/`. Last update is 2 years ago.
The default theme works.
The TA update seems break something :-)

**Comment by orbitalquark** [comment]
11.5 introduces many new theme styles that custom themes will need to add support for. Until then, you can take a look at Textadept's default themes and add definitions for the new styles[1] to your base16 theme.

Sorry for the inconvenience :(

[1]: https://github.com/orbitalquark/textadept/blob/008f9821dbea01b8b2ed120936b3ef017f3895b7/themes/light.lua#L40

**Reply by rgieseke** [reply]
Thanks for the pointer, i had already created an issue: https://github.com/rgieseke/base16-textadept/issues/11

**Comment by rgieseke** [comment]
I've updated the Base16 themes for TA12: https://github.com/rgieseke/base16-textadept
